br/pkg/storage/parse.go

```diff
@@ -140,7 +140,7 @@ func ExtractQueryParameters(u *url.URL, options interface{}) {
 					field.SetBool(v)
 				}
 			case reflect.String:
-				field.SetString(param)
+				field.SetString(strings.TrimSuffix(param, "/"))
```

Comment:
😅

this code is also used by other string fields like `secret-access-key` so we can't place the `TrimSuffix` here.

Reply:
It's embarrassing. Please /cc @kennytm again